synfig-core/src/modules/lyr_std/sphere_distort.cpp

```diff
@@ -163,7 +163,7 @@ Layer_SphereDistort::get_param_vocab()const
 
 	ret.push_back(ParamDesc("clip")
 		.set_local_name(_("Clip"))
-		.set_description(_("When cheked, the area outside the Radius are not distorted"))
+		.set_description(_("When checked, the area outside the Radius are not distorted"))
```

Comment:
need to revert back

Reply:
Why?

Reply:
I think it will be easier to fix .po files if this changes will be in a separate commit.

Reply:
Ah, I see. No, that doesn't makes any influence. ^__^